app/store/middleware.js

```diff
@@ -341,7 +341,7 @@ function cleanupState(action, keepCurrent = false) {
     nextState.errors = payload.errors;
 
     return {
-        type: 'persist/REHYDRATE',
+        type: action.type,
```

Comment:
I believe this was on purpose, can't remember why.. do you know if this has some secondary effects?

Btw you have build failures

Reply:
I’ll test the PR soon and let you know